metomi/isodatetime/data.py

```diff
@@ -880,11 +880,11 @@ class TimePoint:
 
     Keyword arguments (usually default to None if not provided):
 
-    expanded_year_digits (default 0) - an agreed-upon number of extra
-        digits to represent the year, beyond the default of 4. For example,
+    num_expanded_year_digits (default 0) - an agreed-upon number of extra
```

Comment:
I don't have a preference for `expanded_year_digits` or `num_expanded_year_digits`, but as this property is changing for `TimePoint`, should we change it in the rest of the code too? `grep`'ing for `expanded.year.digit`, looks like `test__00.py`, `parser_spec.py`, `data.py`, `parsers.py`, `dumpers.py`, and `README.md` mention expanded year digits.

Are those reference OK, in which case this rename here is necessary as this property doesn't represent TimePoint's expanded year digits? (not sure if that makes sense, sorry :laughing: ), or are we going to rename them too later?

Reply:
It's the difference between the number of digits and the actual digits themselves 

https://github.com/metomi/isodatetime/blob/2b908adb4745cc3e5ff8a16a62fbb9c75b4348d6/metomi/isodatetime/data.py#L1110-L1111

https://github.com/metomi/isodatetime/blob/2b908adb4745cc3e5ff8a16a62fbb9c75b4348d6/metomi/isodatetime/data.py#L1233-L1237

(not sure why those two are showing different indentations, they're the same)

Before, we had `TimePoint.expanded_year_digits` which was the number of digits, and `TimePoint.get("expanded_year_digits")` which was the digits themselves. The former should probably have always been called `num_expanded_year_digits`, whereas the latter has not been renamed (just converted to `@property`)